aws/app/lambda/reliability/lib/notifyProcessing.js

```diff
@@ -4,78 +4,86 @@ const { extractFileInputResponses, updateTTL } = require("dataLayer");
 const { retrieveFilesFromReliabilityStorage } = require("s3FileInput");
 
 module.exports = async (submissionID, sendReceipt, formSubmission, language, createdAt) => {
-  const templateID = process.env.TEMPLATE_ID;
-  const notify = new NotifyClient("https://api.notification.canada.ca", process.env.NOTIFY_API_KEY);
-  const emailBody = convertMessage(formSubmission, submissionID, language, createdAt);
-  const messageSubject =
-    language === "fr"
-      ? formSubmission.form.emailSubjectFr
-        ? formSubmission.form.emailSubjectFr
-        : formSubmission.form.titleFr
-      : formSubmission.form.emailSubjectEn
-      ? formSubmission.form.emailSubjectEn
-      : formSubmission.form.titleEn;
+  try {
+    // Making sure currently processed submission has valid submission parameters
+    if ((formSubmission.submission === null) || (formSubmission.submission.email === "")) {
```

Comment:
Wondering if we should be checking for the existence of the email property and it's value instead of null?
```js
if(!formSubmission.submission?.email || formSubmission.submission.email === "") throw Error();
```

Reply:
New condition has been implemented